ow_power_system/src/power_system_node.cpp

```diff
@@ -242,9 +246,24 @@ void PowerSystemNode::injectFault (const string& fault_name,
 
   if (!fault_activated && fault_enabled)
   {
-    ROS_INFO_STREAM(fault_name << " activated!");
-    index = 0;
-    fault_activated = true;
+    // Only activate the fault if no other fault has been activated in the simulation thus far.
+    if (one_fault_injected)
+    {
+      ROS_WARN_STREAM_ONCE
+        (fault_name << ": cannot activate; only one fault activation per simulation is supported."
+         << " Any additional faults for the remainder of this simulation will not be injected,"
+         << " even if they are shown as being activated.");
```

Comment:
I suggest a shorter and more informative message, something like "Ignoring activation of fault FOO.  Only one power fault is permitted per simulation, and BAR has already been injected."

Note that FOO and BAR might be the same.